### PR TITLE
Coveralls maven plugin can't parse a date

### PR DESCRIPTION
**Pull request by 2686747**
#1032: added timestamp format for coveralls-maven-plugin

### CODE REVIEW

**Comment by karato** [comment]
@2686747 Thanks for your pull request, let me find someone who can review it

File: .rultor.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ release:
     git push -f heroku $(git symbolic-ref --short HEAD):master
     git reset HEAD~1
     curl -f --connect-timeout 15 --retry 5 --retry-delay 30 http://www.netbout.com
-    #mvn clean install site-deploy -Psite -P!coveralls --errors --settings ../settings.xml --batch-mode
+    mvn clean install site-deploy -Psite -P!coveralls --errors --settings ../settings.xml --batch-mode
```
Comment (dmzaytsev) [inline .rultor.yml]: @2686747 also please remove `-P!coveralls`

**Comment by dmzaytsev** [comment]
@2686747 let's wait until PM assign someone to review

File: .rultor.yml (hunk 1 above)
Comment (2686747) [inline .rultor.yml]: @dmzaytsev I have to remove whole profile direction or just change to -Pcoveralls?

Comment (dmzaytsev) [inline .rultor.yml]: @2686747 please remove whole profile direction

**Comment by karato** [comment]
@dmzaytsev review this pls (it's a rather short pull request)

**Comment by dmzaytsev** [comment]
@2686747 thanks

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@dmzaytsev thanks, paid, **20 mins** to your account, payment ID is `78664169`, task took 22 hours and 22 mins to complete

extra minutes for review comments (c=5)

+20 added to your rating, at the moment it is: [+3609](http://www.netbout.com/b/35166?open=rating)

**Comment by karato** [comment]
@rultor deploy pls

**Pull request merged**